modules/ocf/manifests/packages.pp

```diff
@@ -16,6 +16,10 @@
   include ocf::packages::shell
   include ocf::packages::ssh
 
+  class { 'ocf::packages::gnupg':
+    stage => first,
+  }
+
```

Comment:
I'd probably move this to the manifest that needs it, e.g. `ocf::apt`` probably? (granted, there are other `apt::key`s, but that's the main place)

There are definitely exceptions but mostly this list is "general" packages rather than things required by other manifests.